ansible/micromosler.yml

```diff
@@ -0,0 +1,219 @@
+- name: NFS Support
+  hosts: nfs
+  tasks:
+     -yum state=latest name=nfs-utils
+
+- name: LDAP
+  hosts: ldap
+  tasks:
+    - yum: state=present name=openldap-servers
+    - yum: state=present name=openldap-clients
+    - lineinfile: dest=/etc/sysconfig/iptables state=present line='-A INPUT -m state --state NEW -m tcp -p tcp --dport 389 -j ACCEPT' insertbefore='^-A INPUT -m state --state NEW .*'
+    - copy:   src={{ mm_home }}/configs/ldap_helper.sh dest=/root/ldap_helper.sh
+   
+    - action: shell chmod a+x /root/ldap_helper.sh
+    - action: shell /root/ldap_helper.sh
+     
+
+
+- name: OpenStack packages and more
+  hosts: openstack
+  tasks:
+    - copy: src={{ mm_home }}/configs/RPM-GPG-KEY-Icehouse-SIG dest=/etc/pki/rpm-gpg/RPM-GPG-KEY-Icehouse-SIG
+    - copy: src={{ mm_home }}/configs/rdo-release.repo dest=/etc/yum.repos.d/rdo-release.repo
+    - yum: name=python-novaclient state=latest
+    - yum: name=python-keystoneclient state=latest
+    - yum: name=python-neutronclient state=latest
+    - yum: name=python-glanceclient state=latest
+    - yum: name=python-heatclient state=latest
+    - yum: name=python-neutronclient state=latest
+    - yum: name=openstack-nova state=present 
+    - yum: name=openstack-neutron-ml2 state=present 
+    - yum: name=openstack-neutron-openvswitch state=present 
+    - service: name=openvswitch state=started
+    - lineinfile: dest=/etc/sysconfig/iptables state=present line='-A INPUT -m state --state NEW -s 172.25.8.0/24 -j ACCEPT' insertbefore='^-A INPUT -m state --state NEW .*'
+    - service: name=iptables state=restarted
+
+    - copy: src={{ mm_home }}/configs/keystonerc dest=/root/.keystonerc
```

Comment:
Really hard to understand the changes here, Why do you comment our lots of lines?